.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -44,5 +40,4 @@ body:
       label: Additional Information
       description: |
         Include any additional information such as logs, screenshots, or scenarios in which the bug occurs in order to facilitates resolving the issue.
-
 # yamllint enable rule:line-length
```

Comment:
kill `# yamllint`

Reply:
Will address in https://github.com/twosixlabs/armory/pull/1708